### ISSUE TITLE
Lower GL/GLES version required for some ARM64 devices

### ISSUE THREAD

**Issue by hrdl-github** [ahrm/sioyek#1101]
Running the development version of sioyek on the Pine64 PineNote (RK3566, MALI G52 2EE) results in a black rendering area. I've been working around this by setting the OpenGL version to 2.1 (instead of 3.3) and using 3.0 es instead of 3.3 core for the shaders: https://gitlab.com/hrdl/pinenote-shared/-/raw/main/patches/sioyek/gl_gles_version.patch

Do you have any suggestions on how to increase compatibility? Mentioning @hmpthcs because they brought this up again earlier this week.

**Comment by ahrm** [comment]
> OpenGL version to 2.1 (instead of 3.3) and using 3.0 es instead of 3.3 core for the shaders

I don't think this is correct. Opengl 2.1 does not support 3.3 shaders so I think the fact that it is working is maybe an undocumented accident in your graphics driver. In the future we will support a non-opengl rendering backend which should work on such devices.
